### ISSUE TITLE
WiFi Enhanced Open in soft AP mode (IDFGH-12437)

### ISSUE THREAD

**Issue by ghost-458** [espressif/esp-idf#13457]
### Is your feature request related to a problem?

Open wifi networks can be insecure to users, since an attacker can eavesdrop on wifi packets in the air. I have an application where I don't want to go through the hassle of distributing a PSK but cannot have eavesdroppers. 

### Describe the solution you'd like.

I'd like support for OWE/Enhanced Open when broadcasting a soft AP.  It already exists for STA mode. 

### Describe alternatives you've considered.

- WPA - I cannot easily distribute the PSK, so this is infeasible 
- Open wifi and HTTPS for this purpose, but I don't want to have to distribute keys and have users install them.

**Comment by nishanth-radja** [comment]
@kevinhikaruevans will wps registrar with PBC help for this use case ?

**Comment by kevinhikaruevans** [comment]
I don't think so unfortunately. I'd really like to have an open network for this use case.

**Comment by kevinhikaruevans** [comment]
Hello, I just wanted to see if this task was on the roadmap and when this might get added into ESP-IDF. Thank you!

**Comment by nachiketkukade** [comment]
@kevinhikaruevans , this is a big feature and it's currently not on the Wi-Fi feature roadmap. The ideal use case here would be to have an AP with Open + OWE mixed mode, so that legacy devices (most common case) can fall back to Open mode. But this requires dual Beaconing which is a big requirement for a SoftAP.
